docs/languages/en/ref/overview.rst

```diff
@@ -4,16 +4,16 @@
 Overview
 ********
 
-Zend Framework 2 is an open source framework for developing web applications and services using *PHP* 5.3+. Zend 
+Zend Framework 2 is an open source framework for developing web applications and services using *PHP* 5.4+. Zend 
 Framework 2 uses 100% `object-oriented`_ code and utilises most of the new features of PHP 5.3, namely 
```

Comment:
Same as above, we are talking about 5.4 then introducing 5.3 features, please rewrite the paragraph rather than blindly changing the version number.